acceptance/framework/config/config.go

```diff
@@ -130,6 +131,29 @@ func (t *TestConfig) HelmValuesFromConfig() (map[string]string, error) {
 	if t.EnableOpenshift {
 		setIfNotEmpty(helmValues, "global.openshift.enabled", "true")
 	}
+	nodeSelectors := map[string]string{
```

Comment:
i think we can add the ones which have to run on linux. but do you think it should be configurable via values  file or hardcoded in the template ? 
coz if its configured  to windows it will break.

Reply:
or should we add it to values.yaml only and mention in the comments that only linux is supported

Reply:
@curtbushko - I think we shouldn't hardcode in this repo. Instead we should update the [tutorial](https://developer.hashicorp.com/consul/tutorials/kubernetes/kubernetes-windows-nodes) and hard code linux node selectors in this file -  
https://github.com/hashicorp-education/learn-consul-k8s-windows/blob/main/self-managed/eks/helm/values-v1.yaml

Reply:
@aahel @absolutelightning I personally think that having a global.windows.enabled in the helm chart is the cleanest user experience. That way it gets set in the helm chart it is correct for the acceptance tests and users. Otherwise, when a user is installing consul-k8s on windows then they will need to know which components work on Windows and which ones do not. They will not know that.

If it is updated in the helm chart then the tutorial doesn't need to be updated much.

Reply:
@curtbushko All the previous work of `windows-alpha` and tls acls - https://github.com/hashicorp/consul-k8s/pull/2371 was done in this way only (node selector approach). Actually we believe when `global.windows.enabled` is true or false, node selector will be set to linux only, so keeping a flag doesn't makes sense. So we can also hardcode linux in the values.yml irrespective of global.windows.enabled flag.
Please share your thoughts on this. @ishustava please let us your take on this as well.

Reply:
@absolutelightning I think I understand. Since the control-plane all runs on linux then a toggle is not needed.

In that case, if they are ALWAYS going to be running on linux then the components below should be set in the helm chart instead. I think you will end up with bugs in a normal helm install as a default helm chart will end up with resources running on the windows node.

```
	"connectInject.nodeSelector":                                "kubernetes.io/os: linux",
		"connectInject.apiGateway.managedGatewayClass.nodeSelector": "kubernetes.io/os: linux",
		"client.nodeSelector":                                       "kubernetes.io/os: linux",
		"acls.nodeSelector":                                         "kubernetes.io/os: linux",
		"global.acls.nodeSelector":                                  "kubernetes.io/os: linux",
		"global.tls.nodeSelector":                                   "kubernetes.io/os: linux",
		"meshGateway.nodeSelector":                                  "kubernetes.io/os: linux",
		"server.nodeSelector":                                       "kubernetes.io/os: linux",
		"syncCatalog.nodeSelector":                                  "kubernetes.io/os: linux",
		"telemetryCollector.nodeSelector":                           "kubernetes.io/os: linux",
		"webhookCertManager.nodeSelector":                           "kubernetes.io/os: linux",
		"ingressGateways.defaults.nodeSelector":                     "kubernetes.io/os: linux",
		"terminatingGateways.defaults.nodeSelector":                 "kubernetes.io/os: linux",
```

Reply:
@curtbushko  have made this change https://github.com/hashicorp/consul-k8s/pull/2731/commits/2a232acd63bd13fedc979c6b6c1bd6277712a0b6

Reply:
@aahel One last thing. Don't forget bats tests for those components that you changed the node selector for.